pyproject.toml

```diff
@@ -1,10 +1,8 @@
 [build-system]
-requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
+requires = ["setuptools>=61.0"]
 
 [project]
-name = "zuko"
```

Comment:
Important metadata on top for `[project]`. Please revert.

Reply:
rebased onto master.